examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OfferingFragment.kt

```diff
@@ -94,7 +94,7 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
                 "Successful purchase, order id: $orderId",
                 Toast.LENGTH_LONG
             ).show()
+            findNavController().navigateUp()
```

Comment:
this fixes a crash if the context is unavailable once the purchase completes (i.e. you navigate up first)